docs/cvl/methods.md

```diff
@@ -748,8 +748,10 @@ supported.
 The function call may also use the special variable `calledContract`, which
 gives the address of the contract on which the summarized method was called.
 This is useful for identifying the called contract in {ref}`wildcard summaries
-<cvl2-wildcards>`.  The `calledContract` keyword is only defined in the `methods`
-block.
+<cvl2-wildcards>`, both for external and _internal_ functions. Note that for
+library functions the `calledContract` is the contract _calling_ the library
+function.
+The `calledContract` keyword is only defined in the `methods` block.
```

Comment:
maybe "may only be used" (more "common speak" i'd say)

Reply:
Done